src/components/FiltersTab/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Button from "../Button";
+import style from "./styles.module.scss";
+
+const FiltersTab = () => {
+  return (
+    <>
+      <div className={style.filters}>
+        <div className={style.categories}>
+          <Button className={style.button} text="ALL" />
```

Comment:
text="ALL" capital might not be good for button text I think. I am normally passing it in lower case and then upper case them with styles. But please google it, may be I am wrong